app/controllers/provider_interface/reports/reference_status_reports_controller.rb

```diff
@@ -0,0 +1,24 @@
+module ProviderInterface
+  module Reports
+    class ReferenceStatusReportsController < ProviderInterfaceController
+      attr_reader :provider
+
+      def show
+        @provider = current_user.providers.find(provider_id)
+        @provider_application_references = @provider
+          .application_references
+          .joins(application_form: :application_choices)
+          .where.not(application_choices: { accepted_at: nil })
+          .select('"references".name, "references".feedback_status,"application_choices".id as application_choice_id, "application_forms".first_name as first_name, "application_forms".last_name as last_name')
```

Comment:
When we return the results I think any references that are "received" should always be at the top of the ordering, wdyt @MylesJarvis ?